### PR TITLE
Newsletter for March 2022

### PR DESCRIPTION
**Pull request by milancurcic**
Let's plan to merge by Monday, March 7.

* [x] GitHub stats data
* [x] PR list
* [x] Flang updates by @AlexisPerry 
* [x] LFortran updates by @certik or @czgdp1807

Thank you!

### CODE REVIEW

**Comment by milancurcic** [comment]
@certik and @czgdp1807 any chance we can get LFortran updates in today? Thank you!

**Comment by certik** [comment]
I am writing it up now, I didn't get to it yesterday, sorry about that.

File: _posts/2022-03-01-Fortran-Newsletter-March-2022.md [hunk 1]
```diff
@@ -0,0 +1,297 @@
...
+* Allow tabs as white space in formats
+* Do not print format tabs
+* Catch I/O of bad derived type at compile time
+* Allow more concurrently open NEWUNIT= values, with recycling
+
+Call notes are recorded and available upon request [here](https://docs.google.com/document/d/10T-S2J3GrahpG4Ooif93NSTz2zBW0MQc_RlwHi0-afY). Please contact Alexis Perry-Holby at aperry@lanl.gov for document access.
+
+### LFortran
+
+TODO @certik
```
Comment (certik) [inline _posts/2022-03-01-Fortran-Newsletter-March-2022.md:252]:
```suggestion
* LFortran is participating in GSoC, please see https://gitlab.com/lfortran/lfortran/-/wikis/GSoC%202022%20Student%20Instructions for instructions how to apply
* 19 Merge Requests merged
* New ASR optimizations
* Addes support for fma (fused-multiply add) in LLVM
* Semantic improves to compile more of stdlib
```

File: _posts/2022-03-01-Fortran-Newsletter-March-2022.md [hunk 2]
```diff
@@ -0,0 +1,301 @@
+---
+layout: post
+title: "Fortran newsletter: March 2022"
+category: newsletter
+author: Milan Curcic, Alexis Perry-Holby
```
Comment (certik) [inline _posts/2022-03-01-Fortran-Newsletter-March-2022.md:5]:
```suggestion
author: Milan Curcic, Alexis Perry-Holby, Ondřej Čertík
```

**Comment by certik** [comment]
Ok, I am done!

File: _posts/2022-03-01-Fortran-Newsletter-March-2022.md [hunk 3]
```diff
@@ -0,0 +1,301 @@
...
+* Allow tabs as white space in formats
+* Do not print format tabs
+* Catch I/O of bad derived type at compile time
+* Allow more concurrently open NEWUNIT= values, with recycling
+
+Call notes are recorded and available upon request [here](https://docs.google.com/document/d/10T-S2J3GrahpG4Ooif93NSTz2zBW0MQc_RlwHi0-afY). Please contact Alexis Perry-Holby at aperry@lanl.gov for document access.
+
+### LFortran
+
+* LFortran is participating in GSoC, please see https://gitlab.com/lfortran/lfortran/-/wikis/GSoC%202022%20Student%20Instructions for instructions how to apply
```
Comment (certik) [inline _posts/2022-03-01-Fortran-Newsletter-March-2022.md:252]: I guess one has to do links like this:

```suggestion
* LFortran is participating in GSoC, please see [GSoC 2022 Student Instructions for LFortran](https://gitlab.com/lfortran/lfortran/-/wikis/GSoC%202022%20Student%20Instructions) for instructions how to apply
```

**Comment by milancurcic** [comment]
#build_preview

**Comment by milancurcic** [comment]
Thank you all. Will merge.

**Comment by milancurcic** [comment]
#delete_preview

**Pull request merged**